lib/public/views/QcFlags/details/qcFlagDetailsComponent.js

```diff
@@ -50,27 +50,23 @@ const verifyButton = (qcFlagDetailsModel) => {
         qcFlag: remoteQcFlag,
         verificationEnabled,
     } = qcFlagDetailsModel;
+
+    const verificationButton = h(
+        'button#verify-qc-flag.btn.btn-primary',
+        !verificationEnabled
+            ? {
+                onclick: () => {
+                    qcFlagDetailsModel.verificationEnabled = true;
+                },
+            }
+            : { disabled: true },
```

Comment:
Please always put a title why it's disabled

Reply:
added